src/networks.json

```diff
@@ -1349,6 +1349,20 @@
     "start": 3369285,
     "logo": "ipfs://QmXgGmxDAW2Mheum3WX7Q52Zi9hvE17Zp1pVtZbcVdThh4"
   },
+  "239": {
+    "key": "239",
+    "name": "AITD Testnet",
+    "chainId": 239,
+    "network": "Testnet",
+    "testnet": true,
+    "multicall": "0x71f63b35f4a90835678fcb3b53207d7b2ae252fa",
+    "rpc": [
+      "http://54.255.14.223:8545"
```

Comment:
Hey, @xuzuguang This RPC doesn't work anymore. ideally, it should be an official node or some known provider node, 

I tried `http://http-testnet.aitd.io` but it is not a full archive node